lib/marked.js

```diff
@@ -1345,25 +1345,36 @@ marked.setOptions = function(opt) {
   return marked;
 };
 
-marked.defaults = {
-  baseUrl: null,
-  breaks: false,
-  gfm: true,
-  headerIds: true,
-  headerPrefix: '',
-  highlight: null,
-  langPrefix: 'lang-',
-  mangle: true,
-  pedantic: false,
-  renderer: new Renderer(),
-  sanitize: false,
-  sanitizer: null,
-  silent: false,
-  smartLists: false,
-  smartypants: false,
-  tables: true,
-  xhtml: false
-};
+marked.getDefaults = function () {
+  var defaults = {
+    baseUrl: null,
+    breaks: false,
+    gfm: true,
+    headerIds: true,
+    headerPrefix: '',
+    highlight: null,
+    langPrefix: 'lang-',
+    mangle: true,
+    pedantic: false,
+    renderer: new Renderer(),
+    sanitize: false,
+    sanitizer: null,
+    silent: false,
+    smartLists: false,
+    smartypants: false,
+    tables: true,
+    xhtml: false
+  };
+
+  var clone = {};
+  for (var opt in defaults) {
+    clone[opt] = defaults[opt];
+  }
+
+  return clone;
```

Comment:
Is the clone still necessary? Can't you `return defaults;` here instead?

Reply:
I see what you are saying